content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -20,21 +22,25 @@ This guide describes how to migrate your organization’s event monitors and das
 
 ## Migration process
 
-1. **Request early access.** Events Explorer is in private beta. [Contact support][3] to request access for your organization. The support team sends you a notification when Events Explorer is enabled for your organization. You can access the new Explorer and other features through the Events menu in the main navigation, alongside the existing Stream.
+### Migrate data configured externally (Terraform, API)
+If you manage your Datadog configuration with external, API-based scripts or tools such as Terraform, update your dashboard and monitor definitions in your configuration file. Otherwise the old query syntax is applied again at the next deployment and overwrites updated queries. Review the syntax changes described below and use the [migration script][3] to see what converted queries look like. [Contact Support][4] for assistance at this stage.
 
-   {{< img src="events/guides/navigation.png" alt="Events navigation" style="width:100%;" >}}
+### Datadog migrates data stored in Datadog servers
```

Comment:
I'd switch the 2 sections are be more precise.
Something along the line of:
1. Monitor and dashboards definitions stored on Datadog servers are automatically updated (no action required)
2. Definitions stored externally (Terraform or API-based configuration tools must update) must be updated (your action is required)